### DISCUSSION TITLE
AMOC strength at 26 N in zstar and hycom1

### DISCUSSION THREAD

**Discussion by gustavo-marques** [NCAR/omwg_dev#29] [General]
Here is the maximum AMOC at 26 N in runs [004](https://github.com/NCAR/omwg_dev/issues/5) and [005](https://github.com/NCAR/omwg_dev/issues/6):

![image](https://github.com/NCAR/omwg_dev/assets/11339137/de074633-a13d-4b2d-a34b-7926a5028007)

![image](https://github.com/NCAR/omwg_dev/assets/11339137/e8bf87af-428d-4e6c-8d06-936c42c1856b)

Five cycles seem insufficient to achieve an "equilibrated state" in both cases. Also, are we happy with the AMOC strength in these runs?

cc'ing: @iangrooms, @fobryan3, @danabasoglu

**Comment by danabasoglu** [comment]
I suggest running both cases a few more cycles. We would want to know how AMOC "equilibrates" in any case.
